app/models/financialDetails/ChargeItem.scala

```diff
@@ -20,13 +20,15 @@ import exceptions.CouldNotCreateChargeItemException
 import models.incomeSourceDetails.TaxYear
 import play.api.libs.json.{Format, Json}
 import services.DateServiceInterface
+import models.financialDetails.TransactionItem
+import models.financialDetails.TransactionType.format
 
 import java.time.LocalDate
 
 case class ChargeItem (
                         transactionId: String,
                         taxYear: TaxYear,
-                        transactionType: ChargeType,
+                        transactionType: TransactionType,
```

Comment:
Just wondering why we changing this field type, I think as this is ChargeItem I would expect this to be of type "ChargeType"

Reply:
ChargeItem is just the type that's used on the ChargeSummary page, and since we now want to be able to show both charges and credits on there, that type needs to be more flexible